tests/test_unittest/test_call_interface.py

```diff
@@ -42,75 +42,6 @@ def stencil(
         field3 = param * field2
 
 
-def test_assert_same_shape():
```

Comment:
Instead of removing the test, it'd bebetter to make it specific to the dawn backends.

Reply:
Very good idea! I'll make that change.